apps/metadata-utils/src/types.ts

```diff
@@ -97,3 +97,10 @@ export type columnValue = string | number | boolean | columnValueObject;
 interface columnValueObject {
   [x: string]: columnValue;
 }
+
+export type IInputValue = string | number | boolean;
```

Comment:
can we merge this with the other similar data type, I believe it is called ColumnValue?

This is special subset.

Reply:
This will be merged. There are multiple branches that are slightly out of sync with each other. I would like to merge one branch, and update the others so the types can be consistent.